product_harmonized_system/views/product_template.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2010-2024 Akretion France (http://www.akretion.com/)
+  @author Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <!-- product.template form view -->
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">hs_code.product.template.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <group name="group_general" position="inside">
+                <field name="hs_code_id" invisible="type == 'service'" />
+                <field
+                    name="origin_country_id"
+                    invisible="type == 'service'"
+                    options="{'no_create': True}"
+                />
+            </group>
+        </field>
```

Comment:
```suggestion
        <field name="inherit_id" ref="intrastat_base.product_template_form_view" />
        <field name="arch" type="xml">
            <field name="is_accessory_cost" position="after">
                <field name="hs_code_id" invisible="type == 'service'" />
                <field
                    name="origin_country_id"
                    invisible="type == 'service'"
                    options="{'no_create': True}"
                />
            </field>
        </field>
```
I don't see a reason why the H.S. Code and Country of Origin shouldn't be on the Intrastat section of the Product Template form view, declared in the `intrastat_base` module: https://github.com/OCA/intrastat-extrastat/blob/17.0/intrastat_base/views/product_template.xml#L8C17-L8C43.
If done, we could update the readme files to let the people know about it.
Any comments about this? @alexis-via @luc-demeyer